spec/spec_helper.rb

```diff
@@ -38,4 +38,4 @@
 end
 
 # TODO: decide if this is okay to do
-include Fixtures
+# include Fixtures
```

Comment:
If we have changed to use the module namespace  in other files, maybe we can remove these two lines.

Reply:
Removed, thx.